src/lint/lint.ts

```diff
@@ -45,10 +45,9 @@ export async function lint(options: Options, config: ProjectConfig) {
   const rules = lintLib.registry.getRules(ruleCodes || lintOptions.rules);
   const filter = new WarningFilter({
     warningCodesToIgnore: new Set(lintOptions.ignoreWarnings || []),
-    minimumSeverity: Severity.WARNING
+    minimumSeverity: options.minSeverity || lintOptions.minSeverity || Severity.WARNING,
```

Comment:
Up to you if you would like to switch it or I can take a stab at it.

Also, it seems the analyzer has an inverse of this Severity to String. Maybe it would make sense to expose those in the Warning model so this doesn't get duplicated somewhere else? Just a thought.

I will add this for now.